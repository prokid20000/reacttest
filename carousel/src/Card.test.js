import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

//smoke test
it("renders without crashing", function () {
  // this is a low-value test, but better than nothing
  render(<Card  caption="Photo by Richard Pasquarella on Unsplash"
    src= "image1"
    currNum= "0"
    totalNum= "3" />);
});

it("snaptest for Card", function() {
  const { container } = render(
    <Card 
      caption="Photo by Richard Pasquarella on Unsplash"
      src= "image1"
      currNum= "0"
      totalNum= "3"
    />
  );

  //snapshot
  expect(container).toMatchSnapshot();
});