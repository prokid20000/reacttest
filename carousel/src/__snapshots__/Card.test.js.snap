// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snaptest for Card 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
      src="image1"
    />
    <small
      class="Card-small"
    >
      Image 
      0
       of 
      3
      .
    </small>
  </div>
</div>
`;

exports[`works when you click on the both arrows 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
      src="image1"
    />
    <small
      class="Card-small"
    >
      Image 
      0
       of 
      3
      .
    </small>
  </div>
</div>
`;
